mage/enterprise.go

```diff
@@ -0,0 +1,30 @@
+/*
+Copyright 2020 Gravitational, Inc.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package mage
+
+import (
+	"os"
+
+	"github.com/gravitational/trace"
+	"github.com/magefile/mage/mg"
+)
+
+type Enterprise mg.Namespace
+
+func hasE() error {
+	if _, err := os.Stat("e/.github"); !os.IsNotExist(err) {
+		return nil
+	}
+	return trace.BadParameter("Enterprise sources are not available.")
```

Comment:
Add a hint on how to make them available? `git submodule init e && git submodule update`. Also, `trace.NotFound` is prob. more appropriate here.